operator-framework/src/test/java/io/javaoperatorsdk/operator/sample/labelselector/LabelSelectorTestCustomResource.java

```diff
@@ -0,0 +1,15 @@
+package io.javaoperatorsdk.operator.sample.labelselector;
+
+import io.fabric8.kubernetes.api.model.Namespaced;
+import io.fabric8.kubernetes.client.CustomResource;
+import io.fabric8.kubernetes.model.annotation.Group;
+import io.fabric8.kubernetes.model.annotation.ShortNames;
+import io.fabric8.kubernetes.model.annotation.Version;
+
+@Group("sample.javaoperatorsdk")
+@Version("v1")
+@ShortNames("lst")
+public class LabelSelectorTestCustomResource
+    extends CustomResource<Void, LabelSelectorTestCustomResourceStatus>
```

Comment:
Why create a status class when it's not used anywhere and empty?

Reply:
Good point, fixed. (I think I had an issue with this in the past when status was void, but not anymore)